IBPSA/BoundaryConditions/BESTEST.mo

```diff
@@ -0,0 +1,1718 @@
+        annotation (Placement(transformation(extent={{68,-56},{88,-36}})));
+      ComRadIsoPerDir Azi315Til90(
+        til=IBPSA.Types.Tilt.Wall,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.SE,
+        rho=rho) "Azimuth = 45 ° SE, Tilt = 0 °"
+        annotation (Placement(transformation(extent={{-58,68},{-78,88}})));
+      ComRadIsoPerDir Azi045Til90(
+        til=IBPSA.Types.Tilt.Wall,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.SW,
+        rho=rho) "Azimuth = 45 SW, Tilt = 90 °"
+        annotation (Placement(transformation(extent={{-58,36},{-78,56}})));
+      ComRadIsoPerDir Azi270Til30(
+        til=0.5235987755983,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.E,
+        rho=rho) "Azimuth = East, Tilt = 30 °"
+        annotation (Placement(transformation(extent={{-58,2},{-78,22}})));
+      ComRadIsoPerDir Azi000Til30(
+        til=0.5235987755983,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.S,
+        rho=rho) "Azimuth = South, Tilt = 0 °"
+        annotation (Placement(transformation(extent={{-58,-28},{-78,-8}})));
+      ComRadIsoPerDir Azi090Til30(
+        til=0.5235987755983,
+        lat=lat,
+        azi=IBPSA.Types.Azimuth.W,
+        rho=rho) "Azimuth = West, Tilt = 0 °"
+        annotation (Placement(transformation(extent={{-58,-56},{-78,-36}})));
+      Utilities.Psychrometrics.X_pTphi x_pTphi
+        annotation (Placement(transformation(extent={{-48,-90},{-68,-70}})));
+      Utilities.Psychrometrics.ToDryAir toDryAir
+        annotation (Placement(transformation(extent={{-76,-92},{-98,-70}})));
+      WeatherData.ReaderTMY3                          weaDat1(
+        pAtmSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        ceiHeiSou=IBPSA.BoundaryConditions.Types.DataSource.Parameter,
+        ceiHei=1650,
+        totSkyCovSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        opaSkyCovSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TDryBulSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TDewPoiSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        TBlaSkySou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        relHumSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        winSpeSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        winDirSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        HInfHorSou=IBPSA.BoundaryConditions.Types.DataSource.File,
+        HSou=IBPSA.BoundaryConditions.Types.RadiationDataSource.File,
+        filNam=Modelica.Utilities.Files.loadResource(
+            "modelica://IBPSA/Resources/Data/BoundaryConditions/WeatherData/Validation/725650.mos"),
+        calTSky=IBPSA.BoundaryConditions.Types.SkyTemperatureCalculation.TemperaturesAndSkyCover)
+        annotation (Placement(transformation(extent={{-10,-10},{10,10}},
+            rotation=90,
+            origin={50,-90})));
+
+      WeatherData.Bus weaBusDew annotation (Placement(transformation(extent={{36,-82},
```

Comment:
I don't see why you need `weaBusDew` and the data reader that outputs the dew point.

Reply:
I guess this was not clear at all, the reason for that data reader it is just to read the Sky temperature using the temperatures and sky cover. I now updated with comments and changed the nomeclature making it similar to the one in the package sky temperature, so it should be more clear.